jetty-core/jetty-quic/jetty-quic-client/pom.xml

```diff
@@ -51,31 +51,29 @@
 
   <profiles>
     <profile>
-      <id>jdk17</id>
+      <id>enable-foreign</id>
       <activation>
-        <jdk>17</jdk>
+        <jdk>22)</jdk>
```

Comment:
Will this mean that building a release for Jetty 12 requires JDK 22 now?

Reply:
If we want to deploy the foreign binding to central, yes.

Also: you just made me notice that `http3.mod` lists a lib dependency on `jetty-quic-quiche-foreign-${jetty.version}.jar`, which I think means that if that jar ends up on the classpath, H3 is configured, and the JVM is earlier than v22, Jetty would fail to start with `UnsupportedClassVersionError` while trying to load the Foreign binding.

Reply:
This actually is a non-issue, the existing code already catches errors and just logs them.

So in that particular case, you'd just get a warning then the JNA binding would be loaded.